Simplified/Accounts.swift

```diff
@@ -25,40 +47,116 @@ let accountSyncEnabledKey        = "NYPLAccountSyncEnabledKey"
   
   let defaults: UserDefaults
   var accounts = [Account]()
-  var currentAccount: Account {
+  
+  var accountsHaveLoaded: Bool {
+    return !accounts.isEmpty
+  }
+  
+  var loadingCompletionHandlers = [(Bool) -> ()]()
+  var accountsAreLoading: Bool {
```

Comment:
Doesn't seem to be used anywhere. Future-proofing?
Feels like checks using `accountsHaveLoaded ` should be paired with `accountsAreLoading` maybe

Reply:
Oh, you're right. I think removing it is the best idea, since I made it okay to call the load method more than once. In the one place that uses `accountsHaveLoaded`, it just wants to call the load method, regardless of whether it's already loading, because it wants to insert its completion block in there.